tests/sensu-write-json.pp

```diff
@@ -0,0 +1,21 @@
+$h = {
```

Comment:
Could you please update the acceptance test to apply this code and check that the file is formatted correctly. You could use diff, md5, however you want to do comparison.